testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -539,13 +537,15 @@ def params_from_base_suite_setup(request):
             if test.rep_call.failed:
                 failed_test_list.append(test.rep_call.nodeid)
         zip_data = suite_cbllog.get_logs_in_zip()
-        suite_log_zip_file = "Suite_test_log_{}.zip".format(str(time.time()))
+        suite_log_zip_file = "Suite_test_log.zip"
```

Comment:
having time stamp will be helpful, can you add the time stamp back?

Reply:
We have the logic for archiving logs for test level as well as test suite level, I added the logic that if logs already archived in the test level I do not archive at the suite level. If we add the time stamp we end up archiving twice .

We do not need it, we have a unique test name to differentiate the test logs.